### ISSUE TITLE
Bulk addition of conferences

### ISSUE THREAD

**Issue by nilesjohnson** [nilesjohnson/conference-list#57]
If someone who currently maintains a conference list wants to start using the site, they want to add all the conferences they know about at once.  We should implement a way to upload these all at once.

Some additional thoughts:

* probably an admin password should be required
* csv is probably the most convenient input format
* don't send any email; use admin key if edits are needed

**Comment by sethjohnson1** [comment]
I agree with your points including CSV, I would take it a step further and just take straight JSON. This is a great tool: http://www.convertcsv.com/csv-to-json.htm - which I have found to be a better option than coding a CSV converter inside cake myself - though not difficult, I think the its better to let the user have a final look-see over the JSON and paste it in to a single textarea.
